audify/qa/nodes/text_quality.py

```
@@ -0,0 +1,542 @@
+        try:
+            raw = item.get_body_content()
+            if not raw:
+                continue
+            # Strip all HTML tags.
+            text = re.sub(r"<[^>]+>", " ", raw.decode("utf-8", errors="replace"))
+            # Collapse whitespace.
+            text = re.sub(r"\s+", " ", text).strip()
+            if text:
+                texts.append(text)
+        except Exception:
+            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)